openfecli/commands/gather.py

```diff
@@ -582,31 +610,31 @@ def gather(results:List[os.PathLike|str],
     The output is a table of **tab** separated values. By default, this
     outputs to stdout, use the -o option to choose an output file.
     """
-    import csv
-
     # find and filter result jsons
     result_fns = _collect_result_jsons(results)
 
     # pair legs of simulations together into dict of dicts
     legs = _get_legs_from_result_jsons(result_fns, report)
 
-    # compute report and write to output
-    writer = csv.writer(
-        output,
-        delimiter="\t",
-        lineterminator="\n",  # to exactly reproduce previous, prefer "\r\n"
-    )
-    writing_func = {
-        'dg': _write_dg_mle,
-        'ddg': _write_ddg,
-        'raw': _write_raw,
+    # compute report
+    report_func = {
+        "dg": _generate_dg_mle,
+        "ddg": _generate_ddg,
+        "raw": _generate_raw,
     }[report.lower()]
-    writing_func(legs, writer, allow_partial)
+    df = report_func(legs, allow_partial)
+    # write output
+    if isinstance(output, click.utils.LazyFile):
+        df.to_csv(output, sep="\t", lineterminator="\n", index=False)
+
+    # TODO: we can add a --pretty flag if we want this to be optional/preserve backwards compatibility
```

Comment:
I think I rather have a flag that is like `--tsv` that then pints the table out like that and keep rich as the default

Reply:
i think @mikemhenry just means for stdout, not writing to file. (see what I wrote in the description about users possibly using `>>`)